Gemfile

```diff
@@ -9,7 +9,8 @@ gem 'pg', '>= 0.18', '< 2.0'
 gem 'puma', '~> 4.3.1'
```

Comment:
In Gemfile I only changed double quotes with single ones and reorder alphabetically